<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>397</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>60</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/ShortCut/images/login_back.jpg);</string>
  </property>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
background-image: url(:/ShortCut/images/log-in.png);
background-color: rgb(85, 170, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>登  录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);
background-image: url(:/ShortCut/images/log-in.png);
border-radius:10px</string>
   </property>
   <property name="text">
    <string>退  出</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radRemember">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-image: url(:/ShortCut/images/log-in.png);
</string>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ShortCut/images/log-in.png);
color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>340</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ShortCut/images/log-in.png);
color: rgb(255, 85, 0);</string>
   </property>
   <property name="text">
    <string>请遵守使用规范，谢谢合作!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPasswd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-image: url(:/ShortCut/images/log-in.png);
border-color: rgb(85, 170, 255);
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="editUserName">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
background-image: url(:/ShortCut/images/log-in.png);
color: rgb(0, 0, 255);
border-color: rgb(85, 170, 255);

</string>
   </property>
  </widget>
  <widget class="QLabel" name="labPawsswd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 255);
border-radius:10px;
background-image: url(:/ShortCut/images/log-in.png);</string>
   </property>
   <property name="text">
    <string>密        码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labUserName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 255);
border-radius:10px;
background-image: url(:/ShortCut/images/log-in.png);</string>
   </property>
   <property name="text">
    <string>用  户  名：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
